templates/default/slurm/parallelcluster_slurm_cloudbursting.conf.erb

```diff
@@ -0,0 +1,8 @@
+[slurm_cb_config]
+region = us-east-2
+proxy = NONE
+cluster_name = <%= node['cfncluster']['stack_name'].sub("parallelcluster-", "") %>
+max_retry = 3
+update_node_address = True
+max_batch_size = 10
+loglevel = INFO
```

Comment:
I would move the logging configuration (handlers included) to a logging specific configuration file